persistence-cassandra/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/cassandra/testkit/TestUtil.scala

```diff
@@ -3,58 +3,19 @@
  */
 package com.lightbend.lagom.scaladsl.persistence.cassandra.testkit
 
-import akka.persistence.PersistentActor
+import com.lightbend.lagom.internal.persistence.testkit.PersistenceTestConfig.cassandraConfig
+import com.lightbend.lagom.internal.persistence.testkit.{ AwaitPersistenceInit => InternalAwaitPersistenceInit }
 import com.lightbend.lagom.scaladsl.persistence.testkit.AbstractTestUtil
-import com.typesafe.config.{ Config, ConfigFactory }
+import com.typesafe.config.Config
 
+@deprecated("Internal object, not intended for direct use.", "1.5.0")
```

Comment:
so, we deprecate it now and we move it to private internal in 1.5.0? Is that the plan?

Reply:
Deprecate it in 1.5.0, remove it in 2.0.

(The private internal objects were added in this pull request, and the accidentally-public ones now delegate to them.)

Reply:
I just realised that this PR is a spin-off from the 2.7.0 and therefore we may need to have it backported to 1.4.x.

The deprecation should target 1.4.7 instead

Reply:
We're not backporting Play 2.7 to Lagom 1.4.

Reply:
We also should not, in general, deprecate APIs in patch releases.

Reply:
I understood that this PR was Play 2.6.x compatible. 

Anyway, my bad, I'm mixing up things.
The reason we are extracting it is not to merge in 1.4.x, but to advance on 1.5.x and make Play-2.7.0 PR smaller.